### ISSUE TITLE
Suggestion: Winget Package

### ISSUE THREAD

**Issue by cprivitere** [kelleyma49/PSFzf#270]
Just a suggestion to add this to winget. 

This is one of the few powershell/font related holdouts keeping scoop on my system at this point. :)

**Comment by NicTanghe** [comment]
yes other package managers are essentially bloatware at this point.

**Comment by kelleyma49** [comment]
Does anyone have a good example of a PowerShell module being released as a Winget package that I can use as a starting point?
